docs/reference/builder.md

```diff
@@ -484,15 +484,14 @@ To view an image's labels, use the `docker inspect` command.
 
     EXPOSE <port> [<port>...]
 
-The `EXPOSE` instructions informs Docker that the container will listen on the
+The `EXPOSE` instructions informs Docker that the container listens on the
 specified network ports at runtime. Docker uses this information to interconnect
-containers using links (see the [Docker User
-Guide](../userguide/dockerlinks.md) and to determine which ports to expose to the
-host when [using the -P flag](run.md#expose-incoming-ports).
+containers using links (see [Linking containers together](../userguide/dockerlinks.md))
+and to set up port redirection on the host system when [using the -P flag](run.md#expose-incoming-ports).
 
 > **Note**:
-> `EXPOSE` doesn't define which ports can be exposed to the host or make ports
-> accessible from the host by default. To expose ports to the host, at runtime,
+> `EXPOSE` doesn't make the ports of the container accessible to host by
```

Comment:
@mikebrow Good addition I think a note is overkill here.  Also, I think if you move things around a bit, you can send folks off to other places *after* you make your point.

-->https://gist.github.com/moxiegirl/a331397b917174bf399b


## EXPOSE

    EXPOSE <port> [<port>...]

The `EXPOSE` instruction informs Docker that the container listens on the
specified network ports at runtime. `EXPOSE` does not make the port accessible from the host or external network. To do that, you must use the `-p` flag to publish a port. You can expose one port number and publish it externally under another number.   

Docker uses exposed and published ports to interconnect
containers using links (see [Linking containers together](../userguide/dockerlinks.md))
and to set up port redirection on the host system when [using the -P flag](run.md#expose-incoming-ports).